.github/workflows/integration.yml

```diff
@@ -173,7 +173,7 @@ jobs:
     name: ${{ matrix.os }} JDK 8
     strategy:
       matrix:
-        os: [windows-2022, windows-2019]
+        os: [windows-2016, windows-2019, windows-2022]
```

Comment:
I'm not sure if we should add another Windows os for CI since this may impose additional load. Particularly because Windows CI is slower than that of Ubuntu.

Reply:
The jobs will run in parallel, so I think it does not cause the CI builds to take longer. Nonetheless, I removed it because I realized it was [supposed to be deprecated by now](https://github.com/actions/virtual-environments/issues/4312).